### ISSUE TITLE
🐛[BUG]: structural_mechanics error checking world_edges count in add_world_edges

### ISSUE THREAD

**Issue by lnrSmglln** [NVIDIA/physicsnemo#1123]
### Version

1.2.0

### On which installation method(s) does this occur?

Pip

### Describe the issue

in examples/structural_mechanics/deforming_plate/helpers.py line 67 falling to error when no world space finded, because we have 1d zero torch tensor, and second dim was checking in if construction:

if world_edges.shape[1] == 0:
        # No new world edges to add
        return (
            graph,
            graph.edge_attr,
            torch.zeros((0, graph.edge_attr.shape[1]), device=device),
        )

replacement world_edges.shape[1] == 0 with world_edges.size()[0] == 0 helps.

### Minimum reproducible example

```shell

```

### Relevant log output

```shell

```

### Environment details

```shell

```

**Comment by lnrSmglln** [comment]
I also want to note that if it is impossible to add world space edges, then the learning pipeline and the inference part drops, because the function does not return the same dimension of the edge features.

**Comment by mnabian** [comment]
Hi @lnrSmglln, thank you for reporting this issue. The Hybrid MeshGraphNet model is not currently designed to handle cases where the number of world edge features is zero. Could you share more about your use case and why support for this scenario is necessary? In the meantime, you might try increasing the world_edge_radius to avoid situations with no world edges. We will also update the code to raise an explicit error for this case.

**Comment by lnrSmglln** [comment]
Thank you for your answer, @mnabian . In my use case, I have two scenarios where world edges are not present:

1) A scenario where two objects collide. During their free flight phase, there are no world edges.

2) A scenario involving multi-step prediction within a one-step simulation. This is used for better training in nonlinear dynamics, such as a crash.

**Comment by mnabian** [comment]
Thanks for the information. I have opened a feature request [here](https://github.com/NVIDIA/physicsnemo/issues/1243), and we will add this support in a future release. Also, if you are interested, we always welcome contributions!
